doc/robocompdsl.md

```diff
@@ -179,13 +179,8 @@ def compute(self):
 	return True
 ```
 
-Save the file and run cmake
+Save the file
 
-```bash
```

Comment:
It's needed if the component have UI (QT). Can you check it and change the documentation according to it?

Reply:
Sure I'll have a look into it.